rpc-tests/tests/util.js

```diff
@@ -0,0 +1,102 @@
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import { spawn } from 'child_process';
+import chaiAsPromised from 'chai-as-promised';
+import chai from 'chai';
+import { typesBundleForPolkadot } from '@acala-network/type-definitions';
+
+export default { typesBundle: typesBundleForPolkadot };
+
+chai.use(chaiAsPromised);
+
+export const BINARY_PATH = `../target/release/astar-collator`;
+export const SPAWNING_TIME = 120000;
+const WS_PORT = 9944;
+
+export async function wait (milliseconds = 0) {
+	return new Promise((resolve, reject) => {
+		setTimeout(resolve, milliseconds);
+	});
+}
+
+export async function startAstarNode() {
+	const cmd = BINARY_PATH;
+	const args = [
+		`--dev`,
+		`--execution=native`, // Faster execution using native
+		`--no-telemetry`,
+		`--no-prometheus`,
+		`--port=30333`,
+		`--rpc-port=9933`,
+		`--rpc-external`,
+		`--ws-port=${WS_PORT}`,
+		`--ws-external`,
+		`--rpc-cors=all`,
+		`--rpc-methods=unsafe`,
+		`--tmp`,
+	];
```

Comment:
This is for local dev node - what about testing Shiden/Astar runtimes?

Reply:
The purpose of the task was to make testing environment so we can automate the manual tests we do when releasing a new client/runtime. If we cannot do this then I don't see the benefit so not sure why we should merge it?
Can you provide a reason please?

I am aware that we discussed yesterday to break this into multiple tasks but for the bare minimum, for this to provide value to us, we need to be able to run tests for our production networks.

Please push the code that fails and let's understand why it's happening.

Reply:
Fixed the issue. Using astar-dev and shiden-dev runtimes.

run test with 

```
cd rpc-tests
yarn test:astar
yarn test:shiden
![Screenshot from 2022-07-12 15-35-22](https://user-images.githubusercontent.com/874046/178503293-c060dac6-beee-48c7-9798-4da9a8713653.png)

```